### DISCUSSION TITLE
Shortcut keys apply to all open Windows applications apparently

### DISCUSSION THREAD

**Discussion by timboormans** [Eugeny/tabby#9046] [Q&A] [answered]
I was using development software of Jetbrains in which CTRL+Space triggers the code completion. I just started my Tabby try-out recently and today i had the Jetbrains software opened and needed to check SSH so also Tabby was opened.

When I use Jetbrains applications and use CTRL+Space (on Windows) then the active window changes to Tabby. I checked and there is indeed a shortcut key CTRL+Space defined inside Tabby. So i deleted it and now i can use code completion inside the other application again. But this surely is odd behavior. The shortcut keys inside Tabby should not affect other applications running on my system.

**Comment by Eugeny** [comment, marked as answer]
The "toggle window" shortcut is global because it's used to show/focus the Tabby window back when in another application.
